src/igroupadmin.cpp

```diff
@@ -466,14 +466,25 @@ void usageMain() {
         "certain administrative functions.  It can also be used by regular",
         "users to list groups ('lg') and by the admin for all operations.",
         " ",
+        "After making a new group they will themselves manage, a groupadmin's
+        "first action should be to add their own username to the group.",
```

Comment:
Maybe we should leave an example of how to do this, e.g.:
```bash
$ igroupadmin atg my_new_empty_group groupadmin_username
```

Reply:
Or, get really hand-holdy:
```bash
# as groupadmin "labmanager1"
$ igroupadmin mkgroup lab1
$ igroupadmin atg lab1 labmanager1
```

Reply:
done